DirectProgramming/DPC++FPGA/Tutorials/Features/mem_channels/README.md

```diff
@@ -0,0 +1,306 @@
+# Memory Channels 
+This FPGA tutorial demonstrates how to use the `mem_channel` buffer property in
+conjuction with the `-Xsno-interleaving` flag to reduce the area consumed by a
+DPC++ FPGA design.
+
+***Documentation***:  The [DPC++ FPGA Code Samples
+Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html)
+helps you to navigate the samples and build your knowledge of DPC++ for FPGA.
+<br>
+The [oneAPI DPC++ FPGA Optimization
+Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide)
+is the reference manual for targeting FPGAs through DPC++. <br>
+The [oneAPI Programming
+Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general
+resource for target-independent DPC++ programming.
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15; Windows* 10
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04* 
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® FPGA Add-On for oneAPI Base Toolkit 
+| What you will learn               | How and when to use the `mem_channel` buffer property and the `-Xsno-interleaving` flag
+| Time to complete                  | 30 minutes
+
+
+
+## Purpose
+
+This FPGA tutorial demonstrates an example of using the `mem_channel` buffer
+property in conjuction with the `-Xsno-interleaving` flag to reduce the amount
+of resources required to implement a DPC++ FPGA design.
+
+By default, the Intel® oneAPI DPC++ compiler configures each global memory type
+in a burst-interleaved manner where memory words are interleaved across the
+available memory channels. This usually leads to better throughput because it
+prevents load imbalance by ensuring that memory accesses do not favor one
+external memory channel over another. Moreover, burst-interleaving enables you
```

Comment:
Maybe add a link to our documentation that talks about burst interleaving in more detail. I was hoping to see a diagram of it in action, but no need to add one here if there is one in our official docs.

Reply:
Addressed.